index.html

```diff
@@ -1256,31 +1346,80 @@
   // handle gUM error here
 });
 
-// ... later, in the rendering loop ...
+var gl = canvas.getContext("webgl2");
+// Activate the extension to use single component R32F texture format.
```

Comment:
Is that alwasy available in WebGL 2.0? if so I would write 

Activate the standard WebGL 2.0 extension for using single component R32F texture format